app/src/main/java/org/jellyfin/mobile/player/PlayerFragment.kt

```diff
@@ -204,6 +213,9 @@ class PlayerFragment : Fragment() {
      * Handle current orientation and update fullscreen state and switcher icon
      */
     private fun updateFullscreenStateAndSwitcher(configuration: Configuration) {
+        // Do not handle any orientation changes while being in Picture-in-Picture mode
+        if(requireActivity().isInPictureInPictureMode) return
```

Comment:
```suggestion
        if (requireActivity().isInPictureInPictureMode)
            return
```
You could also combine all the calls to `requireActivity()` in this method to use a `with()` block.

Reply:
Wrapped and done!